helm/ffc-demo-claim-service/jenkins-aws.yaml

```diff
@@ -0,0 +1,7 @@
+name: ffc-demo-claim-service
+imagePullSecret: myregistrykey
+container:
+  allowPrivilegeEscalation: true
```

Comment:
Do we have a reason to allow this? I know it came from the first working pipeline but I left it out of the API Gateway and it seems to be working fine. Since it reduces security, I don't think we should keep it anywhere that doesn't really need it.

Reply:
There was an issue with the default permissions in the AWS cluster which required escalated privileges to make internal calls. The `allowPrivilegeEscalation` setting is currently applied to all services in the working end-to-end deployment, and should be investigated in a separate card to tighten up K8s security.

Reply:
If the api gateway build deployment was successfully tested in kubernetes as being able to communicate with the other APIs without escalation then it doesn't need it. Was this done as part of the development/review?